src/app/pages/view-test-result/components/vehicle-details-card/vehicle-details-card.ts

```diff
@@ -185,6 +185,7 @@ export class VehicleDetailsCardComponent {
   getFlattenArray = (data: string[]): string => flattenArray(data);
 
   displayRegistration() {
-    return this.isADI3() || !!this.instructorRegistrationNumber || this.shouldShowDimensions || !!this.vehicleDetails;
```

Comment:
inconsistant use of ! for the different values comparedto original.  please review.  also can this be unit tested